charts/incubator/external-dns/questions.yaml

```diff
@@ -14,7 +14,118 @@ questions:
 
 # Include{containerBasic}
 # Include{containerAdvanced}
-
+# Include{containerAdvanced}
+  - variable: externaldns
+    group: App Configuration
+    label: External-DNS Configuration
+    schema:
+      additional_attrs: true
+      type: dict
+      attrs:
+        - variable: logLevel
+          label: Log Verbosity Level
+          description: How description the logs are, from info to debug.
+          schema:
+            type: string
+            default: info
+              enum:
+                - value: info
+                  description: info
+                - value: warning
+                  description: warning
+                - value: error
+                  description: error
+                - value: panic
+                  description: panic
+                - value: debug
+                  description: debug
+                - value: fatal
+                  description: fatal
+        - variable: logFormat
+          label: Log Foramt
+          description: Output logs in either text or Json
+          schema:
+            type: string
+            default: text
+              enum:
+                - value: text
+                  description: text
+                - value: json
+                  description: json
+        - variable: interval
+          label: The interval for DNS updates.
+          description: How often the DNS will you update your DNS records
+          schema:
+            type: string
+            default: 1m
+        - variable: provider
+          label: DNS Provider
+          description: DNS provider where the DNS records will be created, eg Cloudflare, Digitalocean or things such as pihole. For others refer to upstream documents
```

Comment:
mind the use of caps for the names here!

Reply:
Roger, I'll fix, trying too literally to follow upstream's docs

<img width="804" alt="image" src="https://github.com/truecharts/charts/assets/89483932/32260602-32d7-4679-b752-ffdf2573fd76">

Reply:
Preferably due to typo risk an enum would be prefered tbh.

Reply:
saves us another future support ticket.

Reply:
Sounds good, didn't see it before last commit, I'll create an enum list for some providers, but there's like 20 of them, so def not all of them

Reply:
Feel free to dunk in all of them.
Users will want them all.

Reply:
I just hope users know how to figure out their provider lol, it's a solid list

```akamai,alibabacloud,aws,aws-sd,azure,azure-dns,azure-private-dns,bluecat,civo,cloudflare,coredns,designate,digitalocean,dnsimple,dyn,exoscale,gandi,godaddy,google,ibmcloud,infoblox,inmemory,linode,ns1,oci,ovh,pdns,pihole,plural,rcodezero,rdns,rfc2136,safedns,scaleway,skydns,tencentcloud,transip,ultradns,vinyldns,vultr```